scripts/setup-macos.sh

```diff
@@ -86,23 +90,33 @@ function cmake_install {
   mkdir -p "${BINARY_DIR}"
   cmake -Wno-dev -B"${BINARY_DIR}" \
     -GNinja \
+    -DCMAKE_INSTALL_PREFIX="$INSTALL_DEPS_PREFIX" \
     -DCMAKE_CXX_STANDARD=17 \
-    "${INSTALL_PREFIX+-DCMAKE_PREFIX_PATH=}${INSTALL_PREFIX-}" \
```

Comment:
This script already used the `INSTALL_PREFIX` variable to allow the user to specify where libraries could be installed. They would then pass the same path to `CMAKE_PREFIX_PATH` to use their custom location.

Maybe we can address this with better documentation? IMO hard-coded absolute prefix paths in CMake are a bit of a code smell.